step1/vanilla-todo/js/views/TodoInputFormView.js

```diff
@@ -0,0 +1,23 @@
+import { qs } from "../helpers.js";
+import View from "./View.js";
+
+export default class TodoInputFormView extends View {
+  constructor() {
+    const tag = TodoInputFormView.name;
+    console.log(`[${tag}] constructor`);
+    super(qs("#todoInput"));
+
+    this.inputElement = qs("[type=text]", this.element);
+    this.bindEvents();
+  }
+
+  bindEvents() {
+    this.on("submit", (e) => this.handleSubmit(e));
+  }
+  handleSubmit(e) {
+    e.preventDefault();
+    const { value } = this.inputElement;
+    this.emit("@submit", { value });
+    this.inputElement.value = "";
+  }
```

Comment:
```suggestion
  bindEvents() {
    this.on("submit", this.handleSubmit);
  }
  handleSubmit = (e) => {
    e.preventDefault();
    const { value } = this.inputElement;
    this.emit("@submit", { value });
    this.inputElement.value = "";
  }
```
이렇게 arrow function을 사용해도 좋을 것 같아요!